### DISCUSSION TITLE
external website - jQuery not working

### DISCUSSION THREAD

**Discussion by theoreaves** [NativePHP/laravel#308] [Q&A] [answered]
So, the project that I am working on is just a basic website wrapper.  I am opening a window, and redirecting to my application that run on a web site.  All seems to work well, except that I have one thing (a select2 dropdown component) that requires jQuery.  jQuery loads fine in the dev tools, but doing a check where it is used, the console says jQuery is not defined.  

My guess is that electron is maybe blocking something?  For what it is worth, I do have an older electron app that does basicly the same thing, and it works fine from there.

**Comment by simonhamp** [comment]
@theoreaves For a basic website wrapper, I'm keen to understand why you're using NativePHP. Wouldn't just using Electron directly be enough for that?

**Reply by theoreaves** [reply]
This is just the first part of my project.  I am also working to develop a api that will do additional things with the web application. But being able to load the remote site is key to the project.

**Comment by theoreaves** [comment, marked as answer]
So, I don't know why this matters, but I was bringing jQuery and select2 in from script tags.  I decided to change my approach, and add compile them in via npm.  And that solved the problem.  I guess I just didn't need to be so lazy about it.  LOL!
